sonar-plugin-api/src/main/java/org/sonar/api/web/DashboardLayout.java

```diff
@@ -23,7 +23,9 @@
  * Possible layouts for a dashboard.
  *
  * @since 2.13
+ * @deprecated since 6.2, as dashboards have been removed
  */
+@Deprecated
 public enum DashboardLayout {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)